rpcclient/wallet.go

```diff
@@ -803,7 +803,7 @@ func (c *Client) PurchaseTicketAsync(fromAccount string,
 		ticketFeeFloat = ticketFee.ToCoin()
 	}
 
-	splitTxVal := uint32(1)
+	splitTxVal := false
```

Comment:
Instead of defaulting to one or the other, since it will also be specified in the wallet config, the JSON null could be used to not explicitly set the option in the RPC parameters, but use the configured option instead.